From 7b75b5febde4b2a9ac93fddbb51a9c33e1638f55 Mon Sep 17 00:00:00 2001
From: Dan Carpenter <dan.carpenter@oracle.com>
Date: Tue, 8 Jul 2014 13:59:07 +0300
Subject: [PATCH 15/61] crypto: qat - remove an unneeded cast
References: FATE#318531
Git-Commit: ac1a2b49ea3dff611d820043a0c597db3867bee0
Patch-Mainline: v3.17

The cast to (unsigned int *) doesn't hurt anything but it is pointless.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/crypto/qat/qat_common/qat_hal.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/qat/qat_common/qat_hal.c b/drivers/crypto/qat/qat_common/qat_hal.c
index 903ca88..28da876 100644
--- a/drivers/crypto/qat/qat_common/qat_hal.c
+++ b/drivers/crypto/qat/qat_common/qat_hal.c
@@ -850,7 +850,7 @@ static int qat_hal_exec_micro_inst(struct icp_qat_fw_loader_handle *handle,
 
 	if ((inst_num > handle->hal_handle->max_ustore) || !micro_inst) {
 		pr_err("QAT: invalid instructs inst_num=%d, micro_inst=0x%p\n ",
-		       inst_num, (unsigned int *)micro_inst);
+		       inst_num, micro_inst);
 		return -EINVAL;
 	}
 	/* save current context */
-- 
2.1.4

